pkg/network/vmispec/interface.go

```diff
@@ -58,6 +58,21 @@ func IsPodNetworkWithMasqueradeBindingInterface(networks []v1.Network, ifaces []
 	return true
 }
 
+func ShiftPrimaryInterface(statusIfaces []v1.VirtualMachineInstanceNetworkInterface, networks []v1.Network) {
```

Comment:
can we use slice logic in order to build the new array ?
something like
statusIfaces[index], statusIfaces[0:index-1], statusIfaces[index+1:len(statusIfaces)-1]

Reply:
fixed

Reply:
Thanks, can we do it with only one append call maybe?
if not maybe split it to two separate lines please

Reply:
One append won't work, it can be split into two lines of cause.